cmd/epp/main.go

```diff
@@ -26,7 +26,7 @@ import (
 
 func main() {
 	// Register all known plugin factories
-	runner.RegisterAllPlgugins()
+	runner.RegisterAllPlugins()
```

Comment:
Can you pls add a comment of an example out-of-tree plugin registration?

Reply:
added